src/entries/popup/components/SideChainExplainer.tsx

```diff
@@ -22,6 +22,8 @@ export const getSideChainExplainerParams = (
   onDismiss: VoidFunction,
 ) => {
   const chainName = chainIdToNameMapping[chainId];
+
+  // FIXME: https://linear.app/rainbow/issue/BACK-1452/provide-locale-copy-from-the-backendnetworks-for-explainers-so-that-we
```

Comment:
fyi this has been on the backend driven todo list for a bit. Not sure if we have a good solution yet, but will raise in stand up on Monday. cc @fringlesinthestreet